### PR TITLE
adding some styles and updates

### LINKED ISSUE #6: Enhance the UI of website
The current UI is not so attractive, we can add new styles and some some formatting

### PR DESCRIPTION
**Pull request by iqbalcodes6602**
Fixes #6 
new color schemes

@Shariar-Hasan please consider this i have made some color updates as u said

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -79,7 +79,7 @@ <h5 class="quote-author" id="quoteAuthorTag">- Steve Jobs</h5>
     <script src="./assets/js/main.js"></script>
     <script>
       var color_selection=document.querySelectorAll('.select_color')
-      var color_array=['#81CACF','#E68398','#7EDE80','rgb(229 229 62)'];
+      var color_array=['rgb(25 116 122 / 30%)','rgb(154 72 89/ 30%)','rgb(95 190 97 / 30%)','rgb(255 255 0 / 30%)'];
```
Comment (Shariar-Hasan) [inline index.html:82]: Can you make the color array element more brighter? i mean more than 30%?

Comment (iqbalcodes6602) [reply index.html:82]: yes i was thinking the same do u mind suggesting me or should i choose on my own?

Comment (Shariar-Hasan) [reply index.html:82]: You can use 50% or 60% or
you can choose your own 😄

Comment (iqbalcodes6602) [reply index.html]: done please check and possible merge.

Comment (Shariar-Hasan) [reply index.html]: got it

**Pull request merged**